File: app/src/main/java/com/example/ase_project_nutrismart/Response/BmiResponse.java
package com.example.ase_project_nutrismart.Response;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

@Entity
public class BmiResponse implements Serializable {

    @PrimaryKey(autoGenerate = true)
    private int id;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public BmiData getData() {
        return data;
    }

    public void setData(BmiData data) {
        this.data = data;
    }

    public BmiData data;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @ColumnInfo(name="message")
    public String message;

}

File: app/src/main/java/com/example/ase_project_nutrismart/Response/BmiResponse.kt
package com.example.ase_project_nutrismart.Response

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.example.ase_project_nutrismart.Response.BmiData
import com.example.ase_project_nutrismart.Response.ExpireData
import com.example.ase_project_nutrismart.Response.Grocery
import com.example.ase_project_nutrismart.Response.RolesResponse
import java.io.Serializable

@Entity
class BmiResponse : Serializable {
    @PrimaryKey(autoGenerate = true)
    var id = 0
    @JvmField
    var data: BmiData? = null

    @ColumnInfo(name = "message")
    var message: String? = null
}